modules/codelite/codelite.lua

```diff
@@ -26,10 +26,20 @@
 		return cfgname
 	end
 
+	-- Element text is not escaped the same as element attributes
+	function codelite.escElementText(value)
+		local result = value:gsub('&', '&amp;')
+		result = result:gsub('<', '&lt;')
+		result = result:gsub('>', '&gt;')
+		return result
+	end
+
 	function codelite.esc(value)
 		local result = value:gsub('&', '&amp;')
 		result = result:gsub('<', '&lt;')
 		result = result:gsub('>', '&gt;')
+		result = result:gsub('"', '\\&quot;')
+		result = result:gsub(' ', '\\ ')
```

Comment:
This is actually isolated entirely to the defines. Leaving this here breaks the debugger selection and the command line arguments.